index.html

```diff
@@ -114,77 +114,119 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-	 
-	      <!-- ________ *Cesar959 card START ________  -->
-	      <div class="card">
-		      <p class="name">Cesar dos Santos de Almeida</p>
-		      <p class="contact">
-		        <a href="https://github.com/Cesar959" target="_blank"><i class="fab fa-github"></i></a>
-		        <a href="https://www.linkedin.com/in/cesarsantosalmeida/" target="_blank"><i class="fab fa-linkedin"></i></a>
-		      </p>
-		      <p class="about">Sou programador júnior, Apaixonado por programação web, formado no curso técnico em informatica do Senac de taubaté.</p>
-		      <div class="resources">
-		      <p>3 Useful Dev Resources</p>
-		      <ul>
-			      <li>
-			        <a href="https://readme.so/" target="_blank" title="Readme.so">
-			          Readme.so
-			        </a>
-			      </li>
-			      <li>
-			        <a href="https://www.frontendmentor.io/challenges" target="_blank" title="Frontend Mentor">
-			          Frontend Mentor
-			        </a>
-			      </li>
-			      <li>
-			        <a href="https://www.dailyui.co/" target="_blank" title="Daily UI">
-			          Daily UI
-			        </a>
-			      </li>
-		      </ul>
-		    </div>
-	    </div>
-	    <!-- ________*Cesar959 card END ________  -->
-        
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->`
into `<!-- ________ YourName Contributor card START ________ -->`